calico-enterprise/operations/monitor/metrics/recommended-metrics.mdx

```diff
@@ -0,0 +1,514 @@
+| Threshold breach symptoms        | Unexpected increases may indicate memory leaks and performance issues with Typha. |
+| Threshold breach recommendations | Check CPU usage on Typha pods and Kubernetes nodes. Increase resources if needed, rollout and restart Typha(s) if needed. |
+| Priority level                   | Optional.                                                    |
+
+### CPU usage
+
+| CPU usage                        |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | *rate(process_cpu_seconds_total[30s]) \* 100*                |
+| Explanation                      | CPU in use by Typha is represented as a percentage of a core.   |
+| Threshold value recommendation   | A spike at startup is normal. It is recommended to achieve a baseline first, then monitor for any unexpected increases from this baseline. A rule of thumb is to investigate maintained CPU usage above 90%. |
+| Threshold breach symptoms        | Unexpected maintained CPU usage could cause Typha to fall behind in updating its clients (e.g. Felix) and could cause delays to policy updates. |
+| Threshold breach recommendations | Check CPU usage on Kubernetes nodes. Increase resources if needed, and rollout restart Typha(s) if needed |
+| Priority level                   | Recommended.                                                 |
+
+### Memory usage
+
+| Memory usage                     |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | *(process_resident_memory_bytes[30s])*                   |
+| Explanation                      | Amount of memory used by Typha.                            |
+| Threshold value recommendation   | It is recommended to achieve a baseline first, then monitor for any unexpected increases from this baseline. A rule of thumb is to investigate if maintained memory usage is above 90% of what is available from the underlying node. |
+| Threshold breach symptoms        | Unexpected maintained memory usage could cause Typha to fall behind in updating its clients (for example, Felix) and could cause delays to policy updates. |
+| Threshold breach recommendations | Check memory usage on Kubernetes nodes. Increase resources if needed, and rollout restart Typha(s) if needed. |
+| Priority level                   | Recommended.                                                 |
+
+### Typha cluster mesh metrics
+
+The following metrics are applicable only if you have implemented cluster mesh.
+
+Note that this metric requires a count syntax because you will have a copy of the metric per RemoteClusterConfiguration. As shown in the table, the value `2 = In Sync` reflects good connections.
+
+remote_cluster_connection_status[cluster="foo"] = 2
```

Comment:
I think we should use code snippet formatting for these.

Also, I think the syntax for a single instance of a metric uses `{}` not `[]`.  Curly braces are for picking out a metric, square braces are for time range queries.

Reply:
Fixed.